Sources/Kevin/Core/Networking/Utils/UserAgent.swift

```diff
@@ -12,7 +12,7 @@ import UIKit
 struct UserAgent {
     
     static var userAgentString: String {
-        "KeviniOSSdk \(Constants.apiVersion)/\(deviceName())/\(deviceVersion())"
+        "KeviniOSSdk/\(Constants.apiVersion) iOSDeviceModel/\(deviceName()) \(deviceVersion())"
```

Comment:
looking at your conversations in slack I think Vaidotas wants only `iOSDeviceModel/arm64` here, which means `UIDevice.current.systemName` is enough? There's no need for `systemVersion` and `deviceName()`

So to answer your question, it would be perfect to reformat:
KeviniOSSdk 2.2.8 -> KeviniOSSdk/2.2.8
arm64 (device) -> as a separate variable under iOSDeviceModel -> iOSDeviceModel/arm64
IOS version should be kept as it is now (no new addition) <------ here, he says don't add this

Reply:
Yes, output of it is `KeviniOSSdk/2.2.8 iOSDeviceModel/arm64 iOS/16.1`

Reply:
is `iOS/16.1` needed? I think not since it's in user agent by default here: `Mozilla/5.0 (iPhone; CPU iPhone OS 15_6 like Mac OS X) Appl...` on android they parse the version from this place in user agent

Reply:
anyways I am approving, since iOS/16.1 can just be ignored if not needed by data team

Reply:
We are using same UserAgent for API request, in those cases it does not have Browser default headers